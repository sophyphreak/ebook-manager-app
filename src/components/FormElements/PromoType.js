import React, { Component } from 'react';
import RadioInput from './RadioInput';

const PromoTypes = (props) => (
  <RadioInput
    options={['Free (Limited Time)', '99 cents', 'Perma Free']}
    {...props}
  />
);

export default PromoTypes;