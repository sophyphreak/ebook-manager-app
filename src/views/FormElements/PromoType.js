import React from 'react';
import RadioInput from './MainElements/RadioInput';
import promoTypeOptions from './options/promoTypeOptions';

const PromoTypes = ({ promoType, onPromoTypeChange, ...props }) => (
  <RadioInput
    options={promoTypeOptions}
    value={promoType}
    onChange={onPromoTypeChange}
    {...props}
  />
);

export default PromoTypes;
