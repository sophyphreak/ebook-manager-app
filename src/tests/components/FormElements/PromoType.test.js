import React from 'react';
import { shallow } from 'enzyme';
import PromoType from '../../../components/FormElements/PromoType';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<PromoType />);
});

test('should render PromoType correctly', () => {
  expect(wrapper).toMatchSnapshot();
});
