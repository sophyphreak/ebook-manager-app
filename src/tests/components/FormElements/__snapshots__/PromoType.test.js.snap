// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PromoType correctly 1`] = `
<RadioInput
  label="Select Promotion Type:*"
  options={
    Array [
      "Free (Limited Time)",
      "99 cents",
      "Perma Free",
    ]
  }
/>
`;
